docs/reference/api/usb/USBSerial.md

```diff
@@ -0,0 +1,32 @@
+## USBSerial
+
+<span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_serial.png)<span>USBSerial class hierarchy</span></span>
+
+You can use the USBSerial interface to emulate a serial port over USB. You can use this serial port as an extra serial port or as a debug solution. It also communicates between Mbed and a computer.
```

Comment:
What are some best practices for using USBSerial? What are some concepts you need to understand to use it? How does this communicate between Mbed and a computer? How is it different from the other USB classes that do this?